cmake/common.cmake

```diff
@@ -374,6 +374,13 @@ function(add_enclave_lib name app_oe_conf_path enclave_sign_key_path)
     if (PBFT)
       target_include_directories(${name} SYSTEM PRIVATE
         ${CCF_DIR}/ePBFT/src/pbft/
+        ${CCF_DIR}/src/ds/
+      )
+    endif()
+    if (PBFT)
```

Comment:
I think we should find out why this was necessary as it is a discrepancy with the virtual enclave version.